.bcr/metadata.template.json

```diff
@@ -0,0 +1,13 @@
+{
+  "homepage": "https://docs.aspect.build/rules/aspect_rules_esbuild",
```

Comment:
Homepage typo

Reply:
I decided the publish-to-bcr is better in a separate pr